modules/custom/d_commerce/modules/d_commerce_products_list/src/Plugin/facets/url_processor/DCommerceProductListUrlProcessor.php

```diff
@@ -38,6 +38,10 @@ public function buildUrls(FacetInterface $facet, array $results) {
         $view = Views::getView($attributes->get('view_id'));
         $view->setDisplay($attributes->get('display_id'));
         $view->setArguments($result->getUrl()->getRouteParameters());
+        $url = $result->getUrl();
+        if ($url->isRouted()) {
+          $view->setArguments($url->getRouteParameters());
```

Comment:
The same is happening in line 40, before the `$url->isRouted()` condition. Is it intentional?

Reply:
What about line 41 @Rashi89? There is the same action as in line 43 but without a condition. Is it intentional?

Reply:
I think we don't have to duplicate this line, so I removed if statement and edited line 40